### DISCUSSION TITLE
Video course keeps logging out

### DISCUSSION THREAD

**Discussion by orbitcoms** [Mair/esp32-course#102] [Course Qeustions]
While I am watching videos I might watch one for 2 videos and then a box comes up requesting to "Register as a Student".

If I log out and back in, I can continue the video watching until it happens again

**Comment by Mair** [comment]
thanks @orbitcoms I'll keep an eye out and see why this would be

**Reply by robbietree8** [reply]
It came across to me too, I need to refresh page to get it working.

**Reply by mikebeepboop** [reply]
This has happened to me as well ever since the content update done 1 or 2 months ago.

**Comment by Mair** [comment]
interesting. How long before you are forced to re-login? Does a refresh resolve it?

**Reply by gmag11** [reply]
Normally pressing F5 is enough

**Comment by Mair** [comment]
I have made an update. Please let me know if the issue persists or is better then before
